### PR TITLE
Add scripts to push latest image, automate this in github action

### PR DESCRIPTION
**Pull request by bogdandrutu**
Signed-off-by: Bogdan Drutu <bogdandrutu@gmail.com>

### CODE REVIEW

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -61,3 +64,6 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           COSIGN_EXPERIMENTAL: true
+
+      - name: Push latest label for core
```
Comment (jpkrohling) [inline .github/workflows/release.yaml:68]: I know this PR is fixing a pain we have, but I'd rather have this as part of goreleaser instead.

**Review by jpkrohling — CHANGES_REQUESTED** [review]

File: .github/workflows/release.yaml (hunk 1 above)
Comment (bogdandrutu) [reply .github/workflows/release.yaml:68]: Then review https://github.com/open-telemetry/opentelemetry-collector-releases/pull/220

Comment (jpkrohling) [reply .github/workflows/release.yaml:68]: Done ;-)

**Comment by codeboten** [comment]
Superseeded by https://github.com/open-telemetry/opentelemetry-collector-releases/pull/220

**Comment by bogdandrutu** [comment]
@codeboten I think the manual script is still good to have, will do a PR with only that

**Comment by bogdandrutu** [comment]
Let's keep it around for some time. One use-case is that if we release `0.70.0` then do a patch release to `0.69.1` that will be getting "latest" which we need then to manually fix.

File: .github/workflows/release.yaml (hunk 1 above)
Comment (bogdandrutu) [reply .github/workflows/release.yaml]: @jpkrohling review again please :)

**Review by codeboten — COMMENTED** [review]
I'm ok with keeping the manual script around in case it's needed. Please add a comment to the file to make it clear that's when this would be used, otherwise i'm sure i'll be going around in 3 months wondering why the hell this file is here and create a PR to delete it 🤣

**Pull request closed without merging**